app/javascript/packages/verify-flow/steps/password-confirm/password-confirm-step.tsx

```diff
@@ -23,6 +47,20 @@ function PasswordConfirmStep({ errors, registerField, onChange }: PasswordConfir
           onChange({ password: event.target.value });
         }}
       />
+
+      {formatHTML(
+        t('idv.forgot_password.link_html', {
+          link: `<button>${t('idv.forgot_password.link_text')}</button>`,
+        }),
+        {
+          button: ({ children }) => (
+            <Button isUnstyled onClick={() => goToForgotPassword()}>
+              {children}
+            </Button>
+          ),
+        },
+      )}
+
```

Comment:
Up to you if we do it here or elsewhere, but for this to match how it exists in the current flow, there's some styling to right-align this, with some extra margin.

```suggestion
      <div className="text-right margin-top-2 margin-bottom-4">
        {formatHTML(
          t('idv.forgot_password.link_html', {
            link: `<button>${t('idv.forgot_password.link_text')}</button>`,
          }),
          {
            button: ({ children }) => (
              <Button isUnstyled onClick={() => goToForgotPassword()}>
                {children}
              </Button>
            ),
          },
        )}
      </div>
```

Reference:

https://github.com/18F/identity-idp/blob/37e8d8b9d31ade5258d35b736970f32338a2d74c/app/views/idv/review/new.html.erb#L34-L39